### ISSUE TITLE
[WORK REQUEST] Merged login / sign-up form and enable to login with Twitter / X to HEROES

### ISSUE THREAD

**Issue by Kisgus** [HeroesHQ/HEROES#176]
🤼‍♂️  **What problem is your work solving.**

I would like a user to login to heroes with twitter in a similiar way as its possible to login with HEROES

🎉  **Describe the work you'd like**

- [ ] Turn https://heroes.build/sign-in & https://heroes.build/sign-up into same "sign-in" page 
- [ ] Login / Signup with Twitter on HEROES on https://heroes.build/sign-in
- [ ] Add Twitter on hunter overview next to Github https://heroes.build/hunters?sort=Completed+Bounties
- [ ] Add Twitter on detailed profile view next to Github https://heroes.build/hunters/hunter/7b4a330ceaec4de6afa3f4a4e9159c9b

**Comment by cudam321** [comment]
Login / Signup with Twitter on HEROES on https://heroes.build/sign-in
https://www.figma.com/file/kqZ5FzeLV1WBAERZCngbFC/Heroes?type=design&node-id=2656-3999&mode=design&t=KK44WGga7d30TYvY-0
<img width="524" alt="image" src="https://github.com/HeroesHQ/HEROES/assets/97611620/a299503f-6b93-441e-ab39-97f4556b63aa">

**Comment by eclipseeer** [comment]
Login with Google and Apple is postponed for now. Google require a detail [app verification](https://support.google.com/cloud/answer/13464321?hl=en&ref_topic=13460882&sjid=12953941926082881140-NA#) to do it.

**Comment by Kisgus** [comment]
![Image](https://github.com/HeroesHQ/HEROES/assets/18598519/d4b679aa-9920-4756-a6ff-706a16368bfb)

Hey @eclipseeer, so when reviewing it it looks like we have alle 3 options live, is that correct?

**Comment by eclipseeer** [comment]
Before we can push it on the prod we need to verify Twitter account and Google account
